### PR TITLE
Polish DG

### PR DESCRIPTION
**Pull request by Justinhoejj**
Mostly punctuation fixes.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -374,29 +371,29 @@ Step 5. CONNECTIONS will display a detailed help message on the usage of `add` c
...
 
 Step 2. The first contact in the birthday reminder list will have the next birth month and day with respect 
 to current day.
 
 Step 3. The rest of the list with birthday after this first contact will be displayed in sorted order. This list is displayed in CONNECTIONS' `UI` in `BirthdayReminderListPanel`.
 
-Step 4. The user decides to scroll through the `BirthdayReminderListPanel`. Once at the end of the list (at contact with latest birthday), cycle back to the contact with the 
-earliest birthday and display remaining contacts in sorted order. 
+Step 4. The user decides to scroll through the `BirthdayReminderListPanel`. After the contact with the latest birthday is displayed, the contact with the 
+earliest birthday and all remaining undisplayed contacts with a birthday will be displayed in sorted order.
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:386]: Can be in one line with the previous line. Maybe saying that birthday is sorted by day and month in chronological order is better?

**Review by Jiwei99 — APPROVED** [review]
LGTM other than one minor comment

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Justinhoejj) [reply docs/DeveloperGuide.md]: Ok thanks, just push a change hopefully its clearer, can help see if acceptable?

**Review by leezhixuan — APPROVED** [review]
LGTM

**Pull request merged**